Index: src/components/main/item/algorithmFilter.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@use \"styles/module\" as c;\n\n// TODO: 위치 수정\n\n.algoritmFilterBtn {\n  @include c.drop-down-btn;\n  @include c.black-shadow;\n  position: absolute;\n  top: -1.25rem;\n  right: 0;\n  z-index: 1;\n  color: c.$black-color;\n  background-color: c.$white-color;\n  text-align: center;\n  width: 6.875rem;\n  font-weight: 700;\n  padding: 0.75rem;\n  li {\n    font-weight: 400;\n  }\n  @media (max-width: c.$laptop) {\n    top: 0;\n  }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/main/item/algorithmFilter.module.scss b/src/components/main/item/algorithmFilter.module.scss
--- a/src/components/main/item/algorithmFilter.module.scss	(revision 0bce8dd98a1e011427a8eac0b75ee78767196259)
+++ b/src/components/main/item/algorithmFilter.module.scss	(date 1653008624165)
@@ -2,7 +2,7 @@
 
 // TODO: 위치 수정
 
-.algoritmFilterBtn {
+.algorithmFilterBtn {
   @include c.drop-down-btn;
   @include c.black-shadow;
   position: absolute;
